multiaccounts/settings/columns.go

```diff
@@ -197,6 +197,11 @@ var (
 		dBColumnName:   "notifications_enabled",
 		valueHandler:   BoolHandler,
 	}
+	NotificationsSettings = SettingField{
+		reactFieldName: "notifications-settings",
+		dBColumnName:   "notifications_settings",
+		valueHandler:   JSONBlobHandler,
+	}
```

Comment:
Ideally we should not store JSON blobs (in fact in the future we should migrate the existing jsons to separate tables)

Reply:
@richard-ramos I agree and I am totally up to that, but I just followed what we had there. If we're talking how it should be, we also shouldn't have new column for each new settings field. How our `settings` table looks now... it has 30+ maybe 40+ columns and a single row, that's all. So that's not the way of doing things, but changing that flow is a big change and should be done in a separate ticket aligning with mobile team.

Reply:
the migration has started: https://github.com/status-im/status-go/pull/2470 also as u said quite complex (and we don t have much time for it). maybe we should not just add new blob.

Reply:
I see it as some of the following:
- accept this PR and deal with this column along with the other columns (since this is not the only blob containing json string)
- to add separate column within this table for each notification settings property we need to store, but there is another issue... we need to store exemptions as well and they refer to `1:1` and `group` chat and `community` and those notifications properties are maintained by their ids

@alaibe wdyt, your suggestion?

Reply:
Another option is to introduce a new table instead of columns here and then created dedicated APIs in go to easily access those settings.

See all `communities_settings` related APIs as an inspiration (did the same thing there as storing the data as BLOB wasn't feasible due to future migration challenges)

Reply:
Ok, so what do you guys think? Should I do that within this PR?

Reply:
I'd say yes (not for all the fields, just for the newly introduced of course), as once it's out, it's difficult to migrate, so best to have it in the shape we want in the first go, what do you think?

Reply:
@cammellos deal, will add these settings to a new table soon.